src/main/java/Task.java

```diff
@@ -0,0 +1,50 @@
+public class Task {
+
+    protected String description;
+    protected boolean isDone;
+
+    protected static int numberOfTask = 0;
```

Comment:
You can consider using plural form since the variable is for multiple tasks i.e. `numberOfTasks`.
```suggestion
    protected static int numberOfTasks = 0;
```